pkgs/applications/misc/inkcut/default.nix

```diff
@@ -18,6 +19,27 @@ buildPythonApplication rec {
     sha256 = "sha256-S5IrNWVoUp1w+P7DrKlOUOyY3Q16CHSct9ndZOB3UpU=";
   };
 
+  patches = [
+    # fix opening the extension on inkscape 1.2
+    # https://github.com/inkcut/inkcut/pull/340
+    (fetchpatch {
+      url = "https://github.com/inkcut/inkcut/commit/d5d5d0ab3c588c576b668f4c7b07a10609ba2fd0.patch";
+      hash = "sha256-szfiOujuV7OOwYK/OU51m9FK6dzkbWds+h0cr5dGIg4=";
+    })
+    # fix loading a document from stdin (as used from the extension)
+    # https://github.com/inkcut/inkcut/issues/341
+    (fetchpatch {
+      url = "https://github.com/raboof/inkcut/commit/e2b618e6e21dc8b27bdbf94d938d29e395e01d0a.patch";
```

Comment:
Probably best to keep this one in-tree if you're not making a PR for it, just in case you delete your fork.

Reply:
Its this a straight revert of https://github.com/raboof/inkcut/commit/748ab4157f87afec37dadd715094e87d02c9c739 ?
Then use the revert flag

Reply:
> Then use the revert flag

Didn't know we had that, cool.

Reply:
that is indeed cool, TIL